app/controllers/autocomplete_controller.rb

```diff
@@ -7,8 +7,11 @@ def members
   end
 
   def lists
+    existing_lists = current_user.group_owned_lists.pluck(:id)
```

Comment:
@ptrikutam Whoops, wrong one - this should be `current_resource.lists.pluck(:id)` because we don't want the lists to which the current resource already belongs to. Fixing it.